src/controllers/issuer.ts

```diff
@@ -388,7 +388,8 @@ export class IssuerController {
 	 *         content:
 	 *           application/json:
 	 *             schema:
-	 *               $ref: '#/components/schemas/DidResult'
+	 *               type: boolean
+	 *               example: true
```

Comment:
Is this actually the result that it returns, i.e., a boolean value of true/false without being actually nested in a JSON key-value? If the answer is yes, we should:

1. On deactivation, the ideal response would be to send back the DID document (like in a normal GET for a DID) since this will have the property `deactivated` set to true. This might require a change in the code logic.
2. If the above is done, then I believe the schema reference will go back to what it was originally.

Reply:
Done! ✅